File: AD-HOC/2473/main.dart
import 'dart:io';

void main() {
  Map<int, String> results = {3: 'terno', 4: 'quadra', 5: 'quina', 6: 'sena'};
  Set<int> your = stdin.readLineSync()!.split(' ').map(int.parse).toSet();
  Set<int> draw = stdin.readLineSync()!.split(' ').map(int.parse).toSet();
  int hits = your.intersection(draw).length;
  print(results[hits] ?? 'azar');
}

File: AD-HOC/2473/main.js
let results = { 3: 'terno', 4: 'quadra', 5: 'quina', 6: 'sena' };

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let your = lines.shift().split(' ').map(Number);
let draw = lines.shift().split(' ').map(Number);
let hits = your.filter(number => draw.includes(number)).length;
console.log(results[hits] || 'azar');
